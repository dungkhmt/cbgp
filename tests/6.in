30 30
30 30
27 3
14 1
28 2
17 1
19 1
5 0
19 10
17 5
25 0
8 16
27 1
8 22
2 4
18 29
25 26
21 1
24 11
27 16
9 16
7 4
6 0
28 29
12 11
21 13
28 23
25 11
18 20
4 0
15 28
5 1
1
5 1 21 1 0 0 14 8 1 1 13 3 4 2 2 2 15 5 14 4 26 3 10 4 12 4 27 1 29 9 29 1 16 4 13 1 35 0 24 3 38 0 25 1 13 5 28 3 11 5 8 4 12 8 18 4 31 0 33 0 