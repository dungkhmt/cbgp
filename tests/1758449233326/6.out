(27, 3), (14, 1), (28, 2), (17, 1), (19, 1), (5, 0), (19, 10), (17, 5), (25, 0), (8, 16), (27, 1), (8, 22), (2, 4), (18, 29), (25, 26), (21, 1), (24, 11), (27, 16), (9, 16), (7, 4), (6, 0), (28, 29), (12, 11), (21, 13), (28, 23), (25, 11), (18, 20), (4, 0), (15, 28), (5, 1), 
numberIntersection=0 
minAngle=1.0098141724345941 
sumAngle=32.51677745734872 
minEdgeLength=1.0 
sumEdgeLength=653.4950340133759 
minDistance=0.26396306930176416 
sumDistance=32299.935443242026 

0.0000000000 1.0098141724 32.5167774573 1.0000000000 653.4950340134 0.2639630693 32299.9354432420
0: (102, 7),
1: (109, 32),
2: (120, 6),
3: (0, 119),
4: (112, 1),
5: (105, 21),
6: (96, 0),
7: (112, 0),
8: (82, 8),
9: (105, 27),
10: (80, 70),
11: (101, 20),
12: (108, 31),
13: (118, 52),
14: (118, 33),
15: (115, 57),
16: (100, 29),
17: (116, 23),
18: (120, 79),
19: (96, 49),
20: (120, 120),
21: (117, 50),
22: (76, 1),
23: (0, 120),
24: (98, 21),
25: (100, 8),
26: (74, 0),
27: (100, 31),
28: (118, 59),
29: (119, 69),
102 7 109 32 120 6 0 119 112 1 105 21 96 0 112 0 82 8 105 27 80 70 101 20 108 31 118 52 118 33 115 57 100 29 116 23 120 79 96 49 120 120 117 50 76 1 0 120 98 21 100 8 74 0 100 31 118 59 119 69 
0: (102, 7), 1: (109, 32), 2: (120, 6), 3: (0, 119), 4: (112, 1), 5: (105, 21), 6: (96, 0), 7: (112, 0), 8: (82, 8), 9: (105, 27), 10: (80, 70), 11: (101, 20), 12: (108, 31), 13: (118, 52), 14: (118, 33), 15: (115, 57), 16: (100, 29), 17: (116, 23), 18: (120, 79), 19: (96, 49), 20: (120, 120), 21: (117, 50), 22: (76, 1), 23: (0, 120), 24: (98, 21), 25: (100, 8), 26: (74, 0), 27: (100, 31), 28: (118, 59), 29: (119, 69), 
numberIntersection=0 minAngle=1.0098141724345941 sumAngle=32.516777457337355 minEdgeLength=1.0 sumEdgeLength=653.4950340133771 minDistance=0.26396306930176416 sumDistance=32299.935443243303 
Time taken: 10.173 s
